### ISSUE TITLE
 ppocr INFO: Initialize indexs of datasets:['./dataset/rec_gt_train.txt'] list index out of range

### ISSUE THREAD

**Issue by piarosebelledelapaz** [PaddlePaddle/PaddleOCR#11877]
i am currently finetuning paddleocr recognition model and when i run the command:
python tools/train.py -c configs/rec/multi_language/rec_german_lite_train.yml -o Global.pretrained_model=./pretrain_models/german_ppocr_mobile_v1.1_rec_train/best_accuracy.pdparams

one thing i noticed is that it says lists index out of range:
ppocr INFO: Initialize indexs of datasets:['./dataset/rec_gt_train.txt'] 
list index out of range

i checked the images and labels, both have same number of annotations and number of images. i used PPOCR label to annotate my dataset and ensured that i also follow the format for the dataset given which is the following,

image_path\tAnnotation\n
image_path\tAnnotation\n

how to fix this error?

**Comment by zhangyubo0722** [comment]
Please ensure that the image paths and labels in the annotation file for training are separated by "\t", and there are no blank lines in the entire txt file.

**Comment by AMALVAASISH** [comment]
i have the same error, but even after making changes to the file contents , the error is occuring
